easybuild/easyconfigs/t/tbb/tbb-2021.3.0-GCCcore-10.3.0.eb

```
@@ -0,0 +1,30 @@
+##
+# Author:    Robert Mijakovic <robert.mijakovic@lxp.lu>
+##
+easyblock = 'CMakeMake'
+
+name = 'tbb'
+version = '2021.3.0'
+
+homepage = 'https://github.com/oneapi-src/oneTBB'
+description = """Intel(R) Threading Building Blocks (Intel(R) TBB) lets you easily write parallel C++ programs that
+ take full advantage of multicore performance, that are portable, composable and have future-proof scalability."""
+
+toolchain = {'name': 'GCCcore', 'version': '10.3.0'}
+
+source_urls = ['https://github.com/oneapi-src/oneTBB/archive/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['8f616561603695bbb83871875d2c6051ea28f8187dbe59299961369904d1d49e']
+
+builddependencies = [
+    ('binutils', '2.36.1'),
+    ('CMake', '3.20.1'),
+]
+
+sanity_check_paths = {
+    'files': ['include/%(namelower)s/%(namelower)s.h'] +
+             ['lib64/lib%%(namelower)s.%s' % SHLIB_EXT],
+    'dirs': ['lib64', 'include']
```

```suggestion
    'files': ['include/%(name)s/%(name)s.h'] +
             ['lib/lib%%(name)s.%s' % SHLIB_EXT],
    'dirs': ['lib', 'include']
```

`name` is enough and the default dir is `lib` and the `lib64` is created by EB. Also the `lib` folder is more important as e.g. embree explicitely searches that and that is what we ensure in the easyblock

---

Resolved